PYME/recipes/base.py

```diff
@@ -905,6 +905,11 @@ def _inject_tables_from_hdf5(self, key, h5f, filename, extension):
             logger.warning('No metadata found, proceeding with empty metadata')
             mdh = MetaDataHandler.NestedClassMDHandler()
         
+        try:
+            events = h5f.root.Events[:]
```

Comment:
This breaks our compatibility with generic HDF5 - One of the premises of our .hdf loading in recipes is that it should be able to load fairly generic HDF5 - e.g. from MATLAB etc .... and that (as opposed to the .h5r format) we should avoid 'special' table naming conventions whenever possible. It is not at all out of the question that someone could use an 'Events' table to store localisations.

It is probably sufficient to a) check for a table called 'Events' and b) check that it has the right signature / column names (which should stop things blowing up if someone has used the 'Events' name for something else).

Reply:
seems like overkill to me, but I'm OK with it. Will try and get to implementing this soon but might be tomorrow

Reply:
I'm also happy to crack on some of these - a lot of this was just notes to myself, some of which might be better transferred to comments. My next step was going to be to pull the branch down locally.